gather_istio

```diff
@@ -85,6 +85,18 @@ function getCRDs() {
   echo ${result}
 }
 
+# getPilotName gets the name of the Pilot instance in that namespace. 
+function getPilotName() {
+  local namespace="${1}"
+
+  #this will pipe stderr to stdout. In the case where istiod fails and pilot succeeds, the correct one will be picked up. In the case where both return an error, then the error is returned
+  pilotName=$(oc get pods -n ${namespace} -lapp=istiod  -o jsonpath="{.items[0].metadata.name}" 2>&1)
```

Comment:
You could have something like `-l 'app in (istiod, pilot)'`. That would simplify this function and make it more easy to read.

https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement

Reply:
That's awesome. Updated. Thanks.